ecco_v4_py/read_bin_llc.py

```diff
@@ -333,8 +335,8 @@ def load_ecco_vars_from_mds(mds_var_dir,
 
 
 #%%
-def read_llc_to_tiles(fdir, fname, llc=90, skip=0, nk=1, nl=1, 
-                      filetype = '>f4', less_output = True):
+def read_llc_to_tiles_xmitgcm(fdir, fname, llc=90, skip=0, nk=1, nl=1, 
```

Comment:
Rather than making a new routine, what do you think about adding an optional input "use_dask" which uses the xmitgcm routine. However, I don't fully understand the overhead that's trying to be avoided. Is it dask related overhead? If so, why don't we add an extra input to `read_llc_to_tiles`, load_to_memory, which if True then after the xmitgcm call runs `data_tiles.load();`. This should do the same thing that `load_binary_array` does. If the overhead is related to reshaping, then see my comments below, we should use the same dimension order throughout the package to reduce how many times an array is reshaped.